src/EventListener/DeserializeListener.php

```diff
@@ -78,7 +82,7 @@ public function onKernelRequest(GetResponseEvent $event)
         }
 
         $context = $this->serializerContextBuilder->createFromRequest($request, false, $attributes);
-        if (false === $context['input_class']) {
+        if (isset($context['input']) && null === $context['input']['class']) {
```

Comment:
I fear that you have to use `array_key_exists` here. Here if input is defined nut not input.class, it will fail.